SU-SWS/stanford-jumpstart-deployer
# Upgrade to token 1.7 to fix php 7.1 error

# Status
- Ready for review

# Needed By (Date)
- Not urgent. When you have time.

# Steps to Test
1. Build a JSA website WITHOUT this patch
2. Set webserver to use php 7.1
3. Visit /admin/structure/types/manage/stanford_news_item
4. See php errors:
```
Warning: Illegal string offset 'data' in _token_token_tree_format_row() (line 210 of /httpdocs/jsa.su.dev/sites/all/modules/contrib/token/token.pages.inc).
Warning: Illegal string offset 'class' in _token_token_tree_format_row() (line 211 of /httpdocs/jsa.su.dev/sites/all/modules/contrib/token/token.pages.inc).
Error: Cannot use string offset as an array in _token_token_tree_format_row() (line 211 of /httpdocs/jsa.su.dev/sites/all/modules/contrib/token/token.pages.inc).
```
5. Checkout token 1.7
6. Clear caches
7. Revisit page and see errors all gone.
8. Profit

# Associated Issues
- All products are effected by this but it is limited to local development at this time.
